infra/ansible/playbooks/operator.yaml

```diff
@@ -1,47 +1,181 @@
+        dest: "/home/{{ ansible_user }}/config/config-operator.yaml"
+      vars:
+        aligned_layer_deployment_config_file_path: "{{ lookup('ini', 'aligned_layer_deployment_config_file_path', file='ini/config-operator.ini') }}"
+        eigen_layer_deployment_config_file_path: "{{ lookup('ini', 'eigen_layer_deployment_config_file_path', file='ini/config-operator.ini') }}"
+        eth_rpc_url: "{{ lookup('ini', 'eth_rpc_url', file='ini/config-operator.ini') }}"
+        eth_rpc_url_fallback: "{{ lookup('ini', 'eth_rpc_url_fallback', file='ini/config-operator.ini') }}"
+        eth_ws_url: "{{ lookup('ini', 'eth_ws_url', file='ini/config-operator.ini') }}"
+        eth_ws_url_fallback: "{{ lookup('ini', 'eth_ws_url_fallback', file='ini/config-operator.ini') }}"
+        ecdsa_private_key_store_path: "{{ lookup('ini', 'ecdsa_private_key_store_path', file='ini/config-operator.ini') }}"
+        ecdsa_private_key_store_password: "{{ lookup('ini', 'ecdsa_private_key_store_password', file='ini/config-operator.ini') }}"
+        bls_private_key_store_path: "{{ lookup('ini', 'bls_private_key_store_path', file='ini/config-operator.ini') }}"
+        bls_private_key_store_password: "{{ lookup('ini', 'bls_private_key_store_password', file='ini/config-operator.ini') }}"
+        aggregator_rpc_server_ip_port_address: "{{ lookup('ini', 'aggregator_rpc_server_ip_port_address', file='ini/config-operator.ini') }}"
+        operator_tracker_ip_port_address: "{{ lookup('ini', 'operator_tracker_ip_port_address', file='ini/config-operator.ini') }}"
+        address: "{{ lookup('ini', 'address', file='ini/config-operator.ini') }}"
+        metadata_url: "{{ lookup('ini', 'metadata_url', file='ini/config-operator.ini') }}"
+        enable_metrics: "{{ lookup('ini', 'enable_metrics', file='ini/config-operator.ini') }}"
+        metrics_ip_port_address: "{{ lookup('ini', 'metrics_ip_port_address', file='ini/config-operator.ini') }}"
+        last_processed_batch_filepath: "{{ lookup('ini', 'last_processed_batch_filepath', file='ini/config-operator.ini') }}"
+
+    - name: Deposit into wETH strategy
+      when: operator_stake.stdout == "0"
+      make:
+        chdir: /home/{{ ansible_user }}/repos/operator/aligned_layer
+        target: operator_deposit_into_strategy
+        params:
+          CONFIG_FILE=/home/{{ ansible_user }}/config/config-operator.yaml
+          STRATEGY_ADDRESS={{ lookup('ini', 'weth_strategy_address', file='ini/config-register-operator.ini') }}
+          AMOUNT=100000000000000000 # 0.1ether
+      environment:
+        PATH: "{{ ansible_env.PATH }}:/usr/bin:/usr/local/go/bin"
 
     - name: Allow access to tcp port 9092 from the VPN
+      become: true
       ufw:
         rule: allow
         port: 9092
         proto: tcp
-        src: 100.0.0.0/24
\ No newline at end of file
+        src: 100.0.0.0/24
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
+
+    - name: Build the Operator
```

Comment:
Build operator should be first above `Deposit into wETH strategy`

Reply:
Done